#!/usr/bin/env python
PACKAGE = "ur5_control"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("F_TRIGGER", double_t, 0, "Force threshold to trigger the release of the object (if active)", 10.0, 0.0, 1000.0)

exit(gen.generate(PACKAGE, "ur5_control", "ReleaseController"))
